src/stylesheets/components/_footer.scss

```diff
@@ -4,7 +4,16 @@
 // GOV.UK Frontend footer adapted for full width
 
 @include govuk-exports("app-footer") {
+  .app-footer {
+    @include govuk-media-query($from: tablet) {
+      display: flex;
+      flex-direction: column;
+      flex: 1 0 auto;
+    }
+  }
+
   .app-width-container--full {
+    min-width: calc(100% - 60px);
```

Comment:
Where does this 60px come from?

Reply:
Assuming this is 100% -30px gutter on either side then centred by the flex properties?